includes/class-woocommerce.php

```diff
@@ -435,11 +435,17 @@ public function includes() {
 			include_once WC_ABSPATH . 'includes/blocks/class-wc-block-library.php';
 		}
 
+		/**
+		 * WCCOM Site.
+		 */
+		include_once WC_ABSPATH . 'includes/wccom-site/class-wc-wccom-site.php';
+
 		/**
 		 * Libraries
 		 */
 		include_once WC_ABSPATH . 'includes/libraries/action-scheduler/action-scheduler.php';
 
+
```

Comment:
@gedex 
Nitpick: the new line seems redundant.

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/24123/commits/e1743bfadb251bd616375b372102c179911aacdb.